src/router/tourists/index.ts

```diff
@@ -1,18 +1,66 @@
-import express, { Request, Response, NextFunction, Router } from "express";
+import express, { Request, Response, Router } from "express";
 import getPlacesFromGoogleByText from "@utils/getPlacesFromGoogleByText";
+import { PrismaClient, Prisma } from "@prisma/client";
 
 const touristsRouter: Router = express.Router();
+const prisma = new PrismaClient();
+
 touristsRouter.use(express.json());
 
-touristsRouter.get("/", (req, res) => {
-  res.send("Hi");
-});
+touristsRouter.post(
+  "/description",
+  async (req: Request, res: Response): Promise<void> => {
+    const { searchText } = req.body;
+
+    try {
+      const googlePlacesResult = await getPlacesFromGoogleByText(searchText);
+
+      const googlePlaceId = googlePlacesResult.placeID;
+      const formattedAddress = googlePlacesResult.formattedAddress;
+      const latitude = googlePlacesResult.location.lat;
+      const longitude = googlePlacesResult.location.lng;
+      const googleMapsUri = googlePlacesResult.googleMapsUri;
+      const websiteUri = googlePlacesResult.websiteUri;
+      const regularOpeningHours = JSON.stringify(
```

Comment:
we need to look inside the google response to find a convenient way of store this information, and not just json